src/geo/gmaps/gmaps-map-view.js

```diff
@@ -82,6 +82,12 @@ var GoogleMapsMapView = MapView.extend({
     MapView.prototype.clean.call(this);
   },
 
+  listenOnce: function (name, callback) {
+    google.maps.event.addListenerOnce(this._gmapsMap, name, function (event) {
```

Comment:
Remove uneccessary function
```javascript
  google.maps.event.addListenerOnce(this._gmapsMap, name, callback);
```

Reply:
This function is used in the tests to wait after map's idle and ensure that projection exists

Reply:
I think @IagoLast refers to the callback fn. There's no need to define an anonymous fn and you can pass `callback` instead.